app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -538,7 +546,11 @@ function AcuantCapture(
       failedImageResubmission: false,
     });
 
-    trackEvent('idv_selfie_image_added', { captureAttempts });
+    trackEvent('idv_selfie_image_added', {
+      captureAttempts,
+      selfieAttempts: selfieAttempts.current,
```

Comment:
thanks for updating to snake case in `analytics_events.rb`! I think we'll have to update it here as well to pass it in correctly.

```suggestion
      selfie_attempts: selfieAttempts.current,
```